ros_installation.md

```diff
@@ -0,0 +1,197 @@
+# Install Gazebo and ROS
+
+This document provides an overview about the options to use different versions
+of ROS in combination with different versions of Gz. It is recommended to read
```

Comment:
nit: Gz -> Gazebo.
Here and below

Reply:
replaced all them [03cd75f](https://github.com/gazebosim/docs/pull/339/commits/03cd75fdd4c1b2f6fa0d0d8895f1168e67233205)